src/apify_client/clients/resource_clients/build.py

```diff
@@ -0,0 +1,46 @@
+from typing import Any, Dict, Optional
+
+from ..._utils import _parse_date_fields, _pluck_data
+from ..base.actor_job_base_client import ActorJobBaseClient
+
+
+class BuildClient(ActorJobBaseClient):
+    """Sub-client for manipulating a single actor build."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the BuildClient."""
+        super().__init__(*args, resource_path='actor-builds', **kwargs)
+
+    def get(self) -> Optional[Dict]:
+        """Return information about the actor build.
+
+        https://docs.apify.com/api/v2#/reference/actor-builds/build-object/get-build
+
+        Returns:
+            The retrieved actor build data
+        """
+        return self._get()
+
+    def abort(self) -> Dict:
+        """Abort the actor build which is starting or currently running and return its details.
+
+        https://docs.apify.com/api/v2#/reference/actor-builds/abort-build/abort-build
+
+        Returns:
+            The data of the aborted actor build
+        """
+        response = self.http_client.call(
+            url=self._url('abort'),
+            method='POST',
+            params=self._params(),
+        )
+        return _parse_date_fields(_pluck_data(response.json()))
```

Comment:
We could actually move this implementation into `ActorJobBaseClient._abort()`, since it will also be used in the run client in exactly the same way.